src/components/LocationAutoSuggest.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PlacesAutocomplete from "react-places-autocomplete";
```

Comment:
Do we really need an external library? We just need to write a fetch since we already have our own UI if I am not wrong.